travis/setup_conda_osx.sh

```diff
@@ -28,6 +28,8 @@ fi
 wget https://repo.continuum.io/miniconda/Miniconda3-${MINICONDA_VERSION}-MacOSX-x86_64.sh -O miniconda.sh
 bash miniconda.sh -b -p $HOME/miniconda
 export PATH="$HOME/miniconda/bin:$PATH"
+conda init bash
+conda activate base
```

Comment:
I've noticed some strange behavior or warnings and have now changed the osx/linux scripts to do:

```
# Create .conda directory before install to workaround conda bug
# See https://github.com/ContinuumIO/anaconda-issues/issues/11148
mkdir $HOME/.conda
bash miniconda.sh -b -p $HOME/miniconda
$HOME/miniconda/bin/conda init bash
source ~/.bash_profile
conda activate base
```

This changes the first call to conda so it uses an exact path. This stops ci-helpers from having to modify the PATH and then immediately re-source the bash profile which will do it anyway.